tcms/signals.py

```diff
@@ -22,8 +22,8 @@
 from django.dispatch import Signal
 
 __all__ = [
-    'post_update',
-    'user_registered',
```

Comment:
Update all occurences of user_registered in docs-strings and/or docs. This is a public user interface that allows overriding behavior.

Also add a BIG FAT note in the 1st line of commit log that these names have changed (backwards incompatible change) and the reason is resolving pylint issues.

Reply:
Actually suffix these with `_SIGNAL` to make it clear what they are.